src/schema/objects/metadata.yaml

```diff
@@ -1847,6 +1925,16 @@ MRAcquisitionType:
   enum:
     - $ref: objects.enums.TwoD.value
     - $ref: objects.enums.ThreeD.value
+MRAcquisitionType__mrs:
+  name: MRAcquisitionType
+  display_name: MR Acquisition Type
+  description: |
+    Type of sequence readout.
+  type: string
+  enum:
+    - 1D
```

Comment:
add an enum for 1D?

Reply:
I think we should as 1D MRS and MRI are both possible (albeit very rarely done). And 1D is a [valid value](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.8.13.4.html#table_C.8-87).